datahub-web-react/src/app/entity/mlModel/profile/MLModelSummary.tsx

```diff
@@ -26,9 +60,63 @@ export default function MLModelSummary() {
         },
     ];
 
+    const formatDate = (timestamp?: number) => {
+        if (!timestamp) return '-';
+        const milliseconds = timestamp < 10000000000 ? timestamp * 1000 : timestamp;
+        return new Date(milliseconds).toISOString().slice(0, 19).replace('T', ' ');
+    };
+
+    const renderTrainingJobs = () => {
+        const lineageTrainingJobs = model?.properties?.mlModelLineageInfo?.trainingJobs || [];
```

Comment:
let me know if this is a valid approach @shirshanka 

would there be any difference between pulling properties > trainingjobs or pulling relationships > trained by?